LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Regions/RegionManager.cs

```diff
@@ -1,201 +1,228 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-namespace LoRaTools.Regions
-{
-    using System.Collections.Generic;
-    using LoRaTools.LoRaPhysical;
-    using LoRaTools.Utils;
-    using LoRaWan;
-    using Microsoft.Extensions.Logging;
-
-    public static class RegionFactory
-    {
-        public static Region CurrentRegion
-        {
-            get; set;
-        }
-
-        /// <summary>
-        /// Tries to get the <see cref="LoRaRegion"/> based on <paramref name="value"/>
-        /// </summary>
-        public static bool TryGetRegion(LoRaRegion value, out Region region)

```

Comment:
I would make this a simple TranslateToRegion or something like that. You not really trying, as the enum is specified and there are only 2 values.

Reply:
good point, I will change the naming. Keep in mind that the enum will grow to ~10 values.